compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposePanel.desktop.kt

```diff
@@ -31,11 +31,21 @@ import java.awt.event.FocusEvent
 import java.awt.event.FocusListener
 import javax.swing.JLayeredPane
 import javax.swing.SwingUtilities.isEventDispatchThread
+import org.jetbrains.skiko.SkiaLayerAnalytics
 
 /**
  * ComposePanel is a panel for building UI using Compose for Desktop.
+ *
+ * @param skiaLayerAnalytics Analytics that helps to know more about SkiaLayer behaviour.
+ * SkiaLayer is underlying class used internally to draw Compose content.
+ * Implementation usually uses third-party solution to send info to some centralized analytics gatherer.
  */
-class ComposePanel : JLayeredPane() {
+class ComposePanel @ExperimentalComposeUiApi constructor(
+    private val skiaLayerAnalytics: SkiaLayerAnalytics,
+) : JLayeredPane() {
+    @OptIn(ExperimentalComposeUiApi::class)
+    constructor() : this(SkiaLayerAnalytics.Empty)
```

Comment:
Previously default constructor was without @OptIn(ExperimentalComposeUiApi::class).
For user's code it can broke compilation.

Reply:
OptIn is not experimental, it allows to use experimental inside. So users won t notice anything.

If we would use optin for experimental from another module or library, then we can accidentaly break binary compatability in the future, but we use the same module experimental.